### DISCUSSION TITLE
Warning

### DISCUSSION THREAD

**Discussion by gotflo** [natintosh/intl-phone-number-input#364] [Q&A]
when I use the intl_phone_number_input-0.7.0+2 package in my flutter 3.0.6 code I get the following error: 

/C:/src/flutter/.pub-cache/hosted/pub.dartlang.org/intl_phone_number_input-0.7.0+2/lib/src/widgets/input_widget.dart:321:20: Warning: Operand of null-aware operation '!' has type 'WidgetsBinding' which excludes null.
- 'WidgetsBinding' is from 'package:flutter/src/widgets/binding.dart' ('/C:/src/flutter/packages/flutter/lib/src/widgets/binding.dart').
package:flutter/…/widgets/binding.dart:1
    WidgetsBinding.instance!.addPostFrameCallback((timeStamp) {

**Comment by faizank09855** [comment]
We can now **DISCARD THE WARNING** and wait until the authors of these packages deal with the new changes in flutter 3 to solve the problem and update their packages. The developers of the packages are now catching up on these changes and there should be new versions of the affected packages soon.

Example of a package with this issue:

`These warnings should not be breaking anything (just causing log spam, for which we apologize; we plan to fix that in the next release, we didn't realize how big of an issue it would be). If your app no longer works, that is unlikely to be related to this issue`

reference:

https://github.com/flutter/flutter/issues/103561#issuecomment-1126416045
